fpga_interchange/examples/chipdb.cmake

```diff
@@ -0,0 +1,235 @@
+function(create_rapidwright_device_db)
+    # ~~~
+    # create_rapidwright_device_db(
+    #    device <common device>
+    #    part <part>
+    #    output_target <output device target>
+    # )
+    # ~~~
+    #
+    # Generates a device database from RapidWright
+    #
+    # If output_target is specified, the output_target_name variable
+    # is set to the generated output_device_file target.
+    #
+    # Targets generated:
+    #   - rapidwright-<device>-device
+
+    set(options output_target)
```

Comment:
`output_target` should be a 1 value arg.

Reply:
Done